src/includes/enriching-events/scopes/with-scope/dart.mdx

```diff
@@ -0,0 +1,22 @@
+<Note>
+
+This feature is available as a Beta. Features available in Beta are still in-progress and may have bugs. We recognize the irony.
+
+This is available in 5.1.0.beta.1 version of the Sentry Dart/Flutter SDK.
+
+</Note>
+
+```dart
+import 'package:sentry/sentry.dart';
+
+Sentry.captureMessage(
+    'This event has an extra tag',
+    withScope: (scope) {
+        // will be tagged with my-tag="my value"
+        scope.setTag('my-tag', 'my value');
+    },
+);
+
+// will not be tagged with my-tag
+Sentry.captureMessage('This event has no extra tag');
```

Comment:
the description in the docs is:

> We also have support for pushing and configuring a scope in one go. This is typically called with-scope or push-scope which is also very helpful if you only want to send data with one specific event. In the following example we are using that function to attach a level and a tag to only one specific error:

so I suggest changing `captureMessage` to `captureException`, as its an error.
also, setting the scope.level.

this might be wrong in the other SDKs and would have to be aligned with the description too, if you copied this example from other SDK, lets raise an issue for that

Reply:
Raised an issue: https://github.com/getsentry/sentry-docs/issues/3601